<launch>

  <!-- <param name="use_sim_time" value="true" /> -->

  <node pkg="tf" type="static_transform_publisher" name="world" args="0 0 0 0 0 0 tello_camera_odom world 100" />



  <!-- <node name="image_muxer" pkg="flock_driver" type="image_muxer.py" output="screen" /> -->

  <!-- <node name="cloud_map_saver" pkg="flock_driver" type="cloud_map_saver.py" output="screen">
    <param name="OUT_FILE_PATH" type="string" value="/home/arkadiros/map_point.txt" />
    <param name="CLOUD_TOPIC_NAME" type="string" value="/orb_slam2_mono/map_points" />
    <param name="TRIGGER_TOPIC_NAME" type="string" value="/save_cloud_trigger" />
    <param name="POSE_TOPIC_NAME" type="string" value="/orb_slam2_mono/pose" />
    <param name="CAMERA_TOPIC_NAME" type="string" value="/orb_slam2_mono/debug_image" />
  </node> -->


 <node name="flock_driver_node" pkg="flock_driver" type="flock_driver.py" output="screen" >
    <param name="network_interface" type="string" value="" />
    <param name="ID" type="string" value="" />


    <!-- <param name="network_interface" type="string" value="" /> -->
    <!-- <param name="TELLO_IP" type="string" value="192.168.43.98" /> -->

    <!-- <remap from="tello0/camera/image_raw" to="rectify/image_mono" /> -->
    <!-- <remap from="/rectify/image_mono" to="tello0/camera/image_mono" /> -->
    <!-- <remap from="tello0/camera/image_raw" to="/image" /> -->

 </node>


<node name="tello_slam_control" pkg="flock_driver" type="tello_slam_control.py" output="screen">
  <param name="~POSE_TOPIC_NAME" type="string" value="/orb_slam2_mono/pose" />
  <param name="ID" type="string" value="" />
</node>


<node name="tello_ui" pkg="flock_driver" type="tello_ui.py" output="screen">
  <param name="ID" type="string" value="" />
  <param name="~POSE_TOPIC_NAME" type="string" value="/orb_slam2_mono/pose" />
</node>

  <!-- <node name="tello_keyboard_node" pkg="flock_driver" type="tello_keyboard.py" output="screen">
      <remap from="/tello/cmd_vel" to="/tello/cmd_vel" />
      <remap from="/tello/takeoff" to="/tello/takeoff" />
      <remap from="/tello/land" to="/tello/land" />
  </node> -->



</launch>
